src/main/java/seedu/address/logic/parser/task/DeleteTaskCommandParser.java

```diff
@@ -29,7 +26,7 @@ public DeleteTaskCommand parse(String args) throws ParseException {
         ArgumentMultimap argMultimap =
                 ArgumentTokenizer.tokenize(args, PREFIX_ROOM_NUMBER, PREFIX_TASK_NUMBER);
 
-        if (!arePrefixesPresent(argMultimap, PREFIX_ROOM_NUMBER, PREFIX_TASK_NUMBER)
+        if (!ParserUtil.arePrefixesPresent(argMultimap, PREFIX_ROOM_NUMBER, PREFIX_TASK_NUMBER)
```

Comment:
Maybe want to change to TaskParserUtil instead of ParseUtil.

Reply:
The current arrangement is like this: the methods that are common to `Patient`, `Room`, and `Task` are placed into `ParserUtil`. Task-specific methods such as `parseDescription` are in `TaskParserUtil`.

Good observation!